content/post/the-future-is-rusty.en.md

```diff
@@ -0,0 +1,71 @@
+by Ivan Ristic, Founder of SSL Labs
+
+I will not add more comments to the above statement as I believe it to be the most accurate description of C language by far. I won't proclaim myself as a C language expert nevertheless I assume all the 'C citizens' here would agree with it.
+
+
+### But sire isn't C++ the next step? Why Rust if we have C++?
+C++, as described by Linus Torvalds, it's a not-so-good programming language(actually he called it horrible and crappy). Or as stated by Donald Knuth, "C++ has a lot of good features, but it has a lot of dirty corners".
+
+Now with the perspective of such awesome people out of the way let me explain why C++ sucks from my view (again I am not an expert).
+
+- It feels like an abstract collage of a slew of languages like ALGOL, Ada, Simula, CLU and C (of course). 
+- It forces OOP ideologies on the programmer in a very ambiguous manner.
+- It overly complicated for little gain in ease of use. And the so-called efficient abstractions make it hard to change code down the line.
+
+[Read here for Linus Torvalds Comments](http://harmful.cat-v.org/software/c++/linus)
+
+[And, here for why C++ sucks](https://news.ycombinator.com/item?id=11147031)
+
+
+### So what makes Rust any better?
+- Rust is a very different language from C++ and probably what can be called the modern man's C.
+- Rust is a language that guarantees to deny memory leaks and data races by compile-time checks without a Garbage Collector, using the concept of Ownership & Borrowing.
+- Rust also has zero-cost abstraction which allows for almost C-like performance.
+
+Call me a fanboy of Rust but here are some stats for the analytical minds to gnaw at, 
+[StackOverFlow Survey for Most Loved Language:](https://insights.stackoverflow.com/survey/2018#most-loved-dreaded-and-wanted) And... the victor is, you guessed it, Rust.
+While C and C++ score really high on the list of most dreaded languages.
+Also if you check you will find that the stats are quite the same for the few years since Rust 1.0 came out and Rust became a stable language.
+
+#### But I heard Go is backed by Google. And it can also give C-like performance. So how come you aren't selling Go?
+Because Rust has the backing of Mozilla group (the people behind Firefox). And Go is extremely opinionated for it to be viable for System Level Programming (in my view). This means that Go is better used for higher level tasks. Also considering it has a garbage collector attached to its hip.
+
+![put-your-trust-in-rust](https://blog.mozilla.org/firefox/files/2017/03/Put-Your-Trust-in-Rust.png)
+
+### So how does Rust provide memory safety?
+> It's elementary my dear Watson.
+Rust has compile-time reference counting and "Ownership and Borrowing". Which allows it to check the code for possible mistakes, and clear up memory without the need for clunky garbage collectors.

```

Comment:
I am hoping this will render right because from here it feels it's all a single quote.

Reply:
![image](https://user-images.githubusercontent.com/19930870/59147359-0177fe00-8a18-11e9-85da-01962f20ba51.png)
Here's a sample.

Reply:
And yes it's wrong so thanks for catching it.